projects/layout/components/chip-group/chip-group.styles.less

```diff
@@ -0,0 +1,40 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+tui-chip-group,
+[tuiChipGroup] {
```

Comment:
Let's only have the attribute selector. I would like to remove this duality everywhere in next major version